src/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -412,6 +396,18 @@ public static StatusScreen getStatus(Town town, CommandSender sender) {
 					.append(Component.newline())
 					.append(Component.text(translator.of("status_hover_click_for_more")))),
 				ClickEvent.runCommand("/towny:town reslist "+ town.getName()));
+			
+			// Plots
+			screen.addComponentOf("plotsnewline", Component.newline());
+			TextComponent text = Component.empty();
+			Map<TownBlockType, Integer> cache = town.getTownBlockTypeCache().getCache(TownBlockTypeCache.CacheType.RESIDENTOWNED);
+			for (TownBlockType type : TownBlockTypeHandler.getTypes().values()) {
+				int value = cache.getOrDefault(type, 0);
+				text = text.append(Component.text(colourKeyValue(translator.of("status_plot_hover", type.getFormattedName()), String.valueOf(value))).append(Component.newline()));
+			}
+			text = text.append(Component.text(translator.of("status_hover_click_for_more")));
+			screen.addComponentOf("plots", colourHoverKey(translator.of("status_plot_string")), 
+				HoverEvent.showText(text), ClickEvent.runCommand("/towny:town plots"));

```

Comment:
```suggestion
			screen.addComponentOf("plots", colourHoverKey(translator.of("status_plot_string")).hoverEvent(HoverEvent.showText(text)).clickEvent(ClickEvent.runCommand("/towny:town plots")));
```

Reply:
I can't change that, colourHoverKey is a String. Other hover/click events are done the same way.

Reply:
Ah right, legacy text